src/test/java/org/llorllale/cactoos/matchers/HasLinesTest.java

```diff
@@ -63,16 +64,18 @@ public void matches() {
      */
     @Test
     public void failed() {
-        this.exception.expect(AssertionError.class);
-        this.exception.expectMessage(
-            String.format(
-                "Expected: Lines are <[Tom, Mike]>%n but was: <[Tom, John]>"
-            )
-        );
         new Assertion<>(
-            "does not match lines that do not contain the given strings",
-            String.format("Tom%nJohn%n"),
-            new HasLines("Tom", "Mike")
+            "must not match lines that do not contain the given strings",
```

Comment:
@victornoel I think I did comment on the repeated "not" here, and after force-push I can't see that in GitHub anymore. If that's the cause (maybe GitHub can't handle a comment on a line that changed?) I would strongly advise against force-pushing instead of adding commits, as this would disrupt `REV` history and make it impossible for `QA` to check that 3 issues were raised.

Reply:
@scristalli yep, I hate the way github works because only the author can resolve comments: in gitlab for example, anybody can mark the comment as resolved and so usually, it is the reviewer that resolve comments, not the PR author. In the future I think I won't resolve any comment anymore to avoid such confusion.

Reply:
@scristalli I've raised the discussion on telegram if you want to join (https://t.me/zerocracy/7175), I think my position is not as strong as I would think :)